### PR TITLE
Add updated url-configuration screenshot

### PR DESCRIPTION
**Pull request by Wizzel1**
## What kind of change does this PR introduce?
Documentation update

## Additional context
The old screenshot was showing an outdated page which might be confusing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -376,7 +376,7 @@ Supabase.initialize(
 
 The redirect callback url should have this format `[YOUR_SCHEME]://[YOUR_AUTH_HOSTNAME]`
 
-![authentication settings page](https://user-images.githubusercontent.com/689843/124574731-f735c180-de74-11eb-8f50-2d34161261dd.png)
+![authentication settings page](https://raw.githubusercontent.com/Wizzel1/supabase-flutter/5ba88b53b6459ab5420876b50275af316b8f62d2/screenshot.png)
```
Comment (Vinzent03) [inline README.md:379]:
```suggestion
![authentication settings page](https://raw.githubusercontent.com/supabase/supabase-flutter/main/screenshot.png)
```

Comment (Wizzel1) [reply README.md]: @Vinzent03 the link you provided shows a 404

Comment (Vinzent03) [reply README.md]: That's because it's not yet merged. Once the file actually exists on the main branch it should work.

**Comment by Vinzent03** [comment]
Can you rename the file to something more descriptive like `deeplink-config.png` and put it under an `images` directory? The url needs to be updated as well.

**Review by dshukertjr — APPROVED** [review]

**Pull request merged**